server.go

```diff
@@ -172,6 +172,17 @@ type Server struct {
 	// non zero RequestConfig field values will overwrite the default configs
 	HeaderReceived func(header *RequestHeader) RequestConfig
 
+	// DenyRequest is called after receiving the Expect 100 Continue Header
```

Comment:
I think `DenyRequest` is not a clear name for this. Maybe just call it `ContinueHandler`?

Reply:
:+1: